### PR TITLE
Display message if APR below threshold - closes #910

### LINKED ISSUE #910: if the estimated APR <0.01%, should block the action and display "insufficient liquidity in the pool"
instead of showing negative numbers - although they are correct

### PR DESCRIPTION
**Pull request by danieledamiani**

### CODE REVIEW

File: packages/tempus-client_v2/src/components/deposit/Deposit.tsx [hunk 1]
```diff
@@ -1118,6 +1136,27 @@ const Deposit: FC<DepositProps> = ({ narrow, chain }) => {
             }
             {tokenEstimateInProgress && <CircularProgress size={14} />}
           </SectionContainer>
+          {estimatedFixedAprBelowThreshold && (
+            <div className="tf__flex-column-center-vh">
+              <Spacer size={20} />
+              <Button color="primary" variant="contained" onClick={() => null} disabled={true}>
+                <Typography variant="button-text" color="inverted">
+                  {getText('insufficientLiquidity', language)}
+                </Typography>{' '}
```
Comment (ShawTim) [inline packages/tempus-client_v2/src/components/deposit/Deposit.tsx:1145]: is it a typo? `{' '}`

File: packages/tempus-client_v2/src/components/deposit/Deposit.tsx [hunk 2]
```diff
@@ -1118,6 +1136,27 @@ const Deposit: FC<DepositProps> = ({ narrow, chain }) => {
...
+              </Button>
+              <Spacer size={15} />
+              <div className="tc__deposit_insufficient-liquidity-message">
+                <div>
+                  <WarnIcon />
+                </div>
+                <Spacer size={15} />
+                <Typography variant="dropdown-text">
+                  <span dangerouslySetInnerHTML={{ __html: getText('insufficientLiquidityMessage', language) }}></span>
+                </Typography>{' '}
```
Comment (ShawTim) [inline packages/tempus-client_v2/src/components/deposit/Deposit.tsx:1155]: same here

**Review by ShawTim — APPROVED** [review]

File: packages/tempus-client_v2/src/components/deposit/Deposit.tsx (hunk 1 above)
Comment (danieledamiani) [reply packages/tempus-client_v2/src/components/deposit/Deposit.tsx:1145]: no, it was made by prettier and I didn't noticed it

File: packages/tempus-client_v2/src/components/deposit/Deposit.tsx (hunk 2 above)
Comment (danieledamiani) [reply packages/tempus-client_v2/src/components/deposit/Deposit.tsx:1155]: same as above

**Pull request merged**